jax/_src/numpy/lax_numpy.py

```diff
@@ -1415,6 +1415,24 @@ def unravel_index(indices, shape):
   idx = clipped_indices % cumulative_sizes[:-1] // cumulative_sizes[1:]
   return tuple(idx)
 
+@_wraps(np.resize)
+def resize(a, new_shape):
+  if isinstance(new_shape, (int, integer)):
+    new_shape = (new_shape,)
+
+  if _any(dim_length < 0 for dim_length in new_shape):
+    raise ValueError("all elements of `new_shape` must be non-negative")
+
+  a = ravel(a)
+
+  new_size = _prod(new_shape)
+  if a.size == 0 or new_size == 0:
+    return zeros_like(a, shape=new_shape)
+
+  repeats = ceil_of_ratio(new_size, a.size)
+  a = concatenate((a,) * repeats)[:new_size]
```

Comment:
I suspect `tile(a, repeats)[:new_size]` will be more efficient, because it's based on broadcasting rather than concatenation.